content/en/blog/2019/monitoring-external-service-traffic/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: "Monitoring blocked and passthrough external service traffic"
+description: "How can you use Istio to monitor blocked and passthrough external traffic."
+publishdate: 2019-09-19
+attribution: Neeraj Poddar (Aspen Mesh)
+keywords: [monitoring,blackhole,passthrough]
+---
+
+## Monitoring blocked and passthrough external service traffic
+
+Understanding, controlling and securing your external service access is one
+of the key benefits that you get from a service mesh like Istio. From a security
+and operations point of view, it is critical to monitor what external service traffic
+is getting blocked as they might surface possible misconfigurations or a
+security vulnerability if an application is attempting to communicate with a
+service that it should not be allowed to. Similarly, if you currently have a
+policy of allowing any external service access, it is beneficial to monitor
+the traffic so you can incrementally add explicit Istio configuration to allow
+access and better security your cluster. In either case, having visibility into this
+traffic via telemetry is quite helpful as it enables you to create alerts and
+dashboards, and better reason about your security posture. This was a highly
+requested feature by production users of Istio and we are excited that the
+support for this was added in release 1.3.
+
+To implement this, the Istio [default
+metrics](/docs/reference/config/policy-and-telemetry/metrics) are augmented with
+explicit labels to capture blocked and passthrough external service traffic.
+This blog will cover how you can use these augmented metrics to monitor all
+external service traffic.
+
+The Istio control plane configures the sidecar proxy with
+predefined clusters called BlackHoleCluster and Passthrough which block or
+allow all traffic respectively. To understand these clusters, let's start with
+what external and internal services mean in the context of Istio service mesh.
+
+### External and internal services
+
+Internal services are defined as services which are part of your platform
+and are considered to be in the mesh. For internal services, Istio control
+plane provides all the required configuration to the sidecars by default.
+For example, in Kubernetes clusters, Istio configures the sidecars for all
+Kubernetes services to preserve the default Kubernetes behavior of all
+services being able to communicate with other.
+
+External services are services which are not part of your platform i.e. services
+which are outside of the mesh. For external services, Istio provides two
+options, first to block all external service access (enabled  by setting
+`global.outboundTrafficPolicy.mode` to `REGISTRY_ONLY`) and
+second to allow all access to external service (enabled  by setting
+`global.outboundTrafficPolicy.mode` to `ALLOW_ANY`). The default option for this
+setting (as of `release-1.3`) is to allow all external service access. This
+option can be configured via `MeshConfig` as explained
```

Comment:
Replace with "...can be configured via [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-OutboundTrafficPolicy-Mode).